src/components/details/ProductDetail.jsx

```diff
@@ -0,0 +1,60 @@
+
+import ProductView from './ProductView'
+import Products from '../Products/Products';
+
+import './ProductDetail.scss'
+
+import { useParams } from 'react-router-dom'
+import { useDispatch, useSelector } from 'react-redux';
+
+import {getProductDetails, getProducts} from "../redux_api/actions/productActions";
+import { useEffect } from 'react';
+
+function ProductDetail() {
+
+  const {id} = useParams();
+  const dispatch = useDispatch();
+
+  const {product} = useSelector(state=>state.getProductDetails);
+  const {products} = useSelector(state=>state.getProducts);
+
+  useEffect(()=>{
+    if(product&& id!==product.id){
+      dispatch(getProductDetails(id))
+      dispatch(getProducts())
+    }
+  },[dispatch,id]);
+
+  return (
+    <div>
+
+      <ProductView product={product}/>  
+
+      <div className='product-features'>
+        <h2>Features & Details</h2>
```

Comment:
Can we use https://www.patternfly.org/components/text pattternfly text component here for all h1 to h6 and p tag ?